src/app/(tabs)/settings.tsx

```diff
@@ -138,7 +170,11 @@ function SystemInfoMenuTarget() {
     });
   }
 
-  return <MenuList items={items} />;
+  const filteredItems = items.filter(
```

Comment:
Shouldn't the `MenuList` component itself do the filtering automatically based on the current platform?